cni/pkg/install/binaries_test.go

```diff
@@ -54,27 +56,49 @@ func TestCopyBinaries(t *testing.T) {
 			srcFiles:      map[string]string{"istio-cni": "cni111", "istio-iptables": "iptables111"},
 			expectedFiles: map[string]string{"istio-cni": "cni111"},
 		},
+		{
+			name:          "binaries prefix",
+			prefix:        "prefix-",
+			srcFiles:      map[string]string{"istio-cni": "cni111", "istio-iptables": "iptables111"},
+			expectedFiles: map[string]string{"prefix-istio-cni": "cni111", "prefix-istio-iptables": "iptables111"},
+		},
 	}
 
-	for _, c := range cases {
+	for i, c := range cases {
 		t.Run(c.name, func(t *testing.T) {
-			srcDir := t.TempDir()
+			srcDir, err := os.MkdirTemp("", fmt.Sprintf("test-case-%d-src-", i))
+			if err != nil {
+				t.Fatal(err)
+			}
+			defer func() {
+				if err := os.RemoveAll(srcDir); err != nil {
+					t.Fatal(err)
+				}
+			}()
```

Comment:
Not sure why we don't use `t.TempDir()` here.

Reply:
I see it came from https://github.com/maistra/istio/blame/maistra-2.3/cni/pkg/install/binaries_test.go#L86 (as a result of the previous rebase).

I will align it with how upstream does that (which is still a mix, but... in favor of simpler `t.TempDir()` call)